src/main/java/com/prototype/booking/api/BookMeetingRoomFlow.java

```diff
@@ -0,0 +1,67 @@
+package com.prototype.booking.api;
+
+import com.prototype.booking.api.referencedata.TimingsRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.integration.dsl.IntegrationFlow;
+import org.springframework.integration.dsl.IntegrationFlows;
+import org.springframework.integration.handler.LoggingHandler;
+import org.springframework.integration.http.HttpHeaders;
+import org.springframework.integration.http.dsl.Http;
+import org.springframework.messaging.support.MessageBuilder;
+
+import javax.persistence.EntityManager;
+
+@Configuration
+@Slf4j
+public class BookMeetingRoomFlow {
+
+    @Autowired
+    TimingsRepository timingsRepository;
+
+    @Autowired
+    BookMeetingRoom.RoomServiceGateway roomServiceGateway;
+
+    @Bean
+    public IntegrationFlow roomsInfoFlow(EntityManager entityManager) {
+        return IntegrationFlows.from(Http.inboundGateway("/rooms")
+                .requestMapping(m -> m.methods(HttpMethod.GET))
+                .errorChannel("globalErrorChannel.input"))
+                .wireTap("loggingFlow.input")
+                .log(LoggingHandler.Level.INFO, this.getClass().getName(), m -> "Retrieving Total Rooms Information")
+                .handle((p,h) -> roomServiceGateway.getRoomsInfo("all")) //Spring Integration currently has no concept of a message without a payload
+                .get();
+    }
+
+    @Bean
+    public IntegrationFlow bookRoomFlow(EntityManager entityManager) {
+        return IntegrationFlows.from(Http.inboundGateway("/bot/booking")
```

Comment:
```suggestion
        return IntegrationFlows.from(Http.inboundGateway("/booking")
```
Please help me to remove **bot** in the readme file also, which I forgot to remove.
Thanks for help.

Reply:
Removed.